src/main/java/com/backbase/ct/bbfuel/configurator/PocketsConfigurator.java

```diff
@@ -96,17 +101,20 @@ public String ingestPocketArrangementForModeOnetoOneAndSetEntitlements(
      * Ingests Pockets for the given user.
      *
      * @param externalUserId External user ID to ingest pockets for.
+     * @return List<Pocket>
```

Comment:
Please add a description of the return value.

Reply:
added